metagraph-plugin/conda_build_dir/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "metagraph-katana" %}
```

Comment:
I would prefer `conda_recipe` instead of `conda_build_dir`. Especially since `build` makes it look like it might be a build directory in the cmake sense, which it isn't.